.ruby-version

```diff
@@ -1 +1 @@
-2.5.0
+2.6.0
```

Comment:
Any reason to upgrade Ruby?

Reply:
My experience with go recently has convinced me we probably ought to stay up to date with our underlying languages, since not doing that cost me at least two days.

Reply:
I'm not as convinced it's an issue for Ruby (since end-users aren't running MM directly for GCP things). But, I have no strong issues with upgrading